language/tools/transaction-replay/src/lib.rs

```diff
@@ -108,6 +127,23 @@ impl LibraDebugger {
     ) -> Result<Option<Version>> {
         self.debugger.get_version_by_account_sequence(account, seq)
     }
+
+    pub fn run_session_at_version<F>(&self, version: Version, f: F) -> Result<ChangeSet>
```

Comment:
I think we talked about this and don't remember now :)
this function is uncalled for now and that is expected right?

Reply:
That is correct. I'm planning to use this function to implement the future debugging session, execute functions/bisect in the PRs to come.